src/deadline/nuke_submitter/deadline_submitter_for_nuke.py

```diff
@@ -112,6 +118,40 @@ def _add_gizmo_dir_to_job_template(job_template: dict[str, Any]) -> None:
     )
 
 
+def _add_ocio_path_to_job_template(job_template: dict[str, Any]) -> None:
+    if "jobEnvironments" not in job_template:
+        job_template["jobEnvironments"] = []
+
+    # This needs to be prepended rather than appended
+    # as it must run before the "Nuke" environment.
+    job_template["jobEnvironments"].insert(
+        0,
+        {
+            "name": "Add OCIO Path to Environment Variable",
+            "script": {
+                "actions": {"onEnter": {"command": "{{Env.File.Enter}}"}},
+                "embeddedFiles": [
+                    {
+                        "name": "Enter",
+                        "type": "TEXT",
+                        "runnable": True,
+                        "data": """#!/bin/bash
+    echo 'openjd_env: OCIO={{Param.OCIOConfigPath}}'
+    """,
+                    }
+                ],
+            },
```

Comment:
I think we can define environment variables directly in OJD instead of with a script: https://github.com/OpenJobDescription/openjd-specifications/wiki/2023-09-Template-Schemas#4-environment

I haven't used it personally, but it looks like it'd do what we want.

Reply:
Nice! Thanks, its much simpler. 🙏